### ISSUE TITLE
Remixd is crashing. 

### ISSUE THREAD

**Issue by MikeMoldawsky** [ethereum/remix-project#3268]
**Remixd is crashing when IDE tries to connect to localhost.**

Behavior:
1. Run `npx remixd` in the folder you want to share.
2. Connect from the remix IDE to localhost (from https://remix.ethereum.org)
3.  In your local computer's terminal, I've got the following WARN `[WARN] You may now only use IDE at https://remix.ethereum.org to connect to that instance
4. It causes `remixd` to crash on your pc
5. Remix IDE gets disconnected (attached image).

Remixd version: `0.6.9` 
OS: `MacOs Monterey 12.1`

![Screen Shot 2022-12-31 at 13 06 28](https://user-images.githubusercontent.com/16150627/210134496-bc38882a-e5b8-43b7-9dfc-0db763352950.png)

**Comment by srpeterseth** [comment]
same for me!

**Comment by jamesbachini** [comment]
I had a similar problem with this and found it was caused by the initialisation of the hardhat background service. A work around is to move hardhat.config.js or hardhat.config.ts to a different filename temporarily while starting remixd

**Comment by bunsenstraat** [comment]
Correct the issue is here: https://github.com/ethereum/remix-project/blob/fa7271286ee68e87817237abe986335dc31ff217/libs/remixd/src/services/hardhatClient.ts#L32 
It tries to watch a folder that does not exist, causing an exception on node 14, but on v16 it just silently fails because of 
```
Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by Aniket-Engg** [comment]
@MikeMoldawsky Can you check if it is fixed with remixd v0.6.10 ?
